### DISCUSSION TITLE
Toleration and labels for prometheus statefulset

### DISCUSSION THREAD

**Discussion by hopik007** [prometheus-operator/prometheus-operator#4190] [Q&A] [answered]
Hi
Because prometheus pod consuming huge of memory  we would like to migrate to the node in AKS with bigger memory sizing.
I have configured toleration and label on prometheus statefulset and because prometheus-operator always reverting this changes I have to stop prometheus-operator. 
This solution is terrible. 

Could you pls advise me how to configure prometheus statefulset with toleration and label without reverting ?

Thanks a lot

**Comment by paulfantom** [comment, marked as answer]
You need to set it up on `Prometheus` CR object. Object docs: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#prometheusspec

**Reply by hopik007** [reply]
I will check. 
Thanks for answer
